node/app.js

```diff
@@ -185,14 +185,19 @@ function receivedMessage(event) {
     var quickReplyPayload = quickReply.payload;
     console.log("Quick reply for message %s with payload %s",
       messageId, quickReplyPayload);
-    if(quickReplyPayload == 'getStartedTalk') {
-      sendQuickReply(senderID, 'getStartedTalk');
-    } else if(quickReplyPayload == 'getStartedAct') {
-      sendQuickReply(senderID, 'getStartedAct');
-    } else if(quickReplyPayload == "getStartedHelp") {
-      sendQuickReply(senderID, 'getStartedHelp');
-    }
 
+    //Checking if quickReply exists 
+  jsonfile.readFile(quickRepliesPath, function(err, obj) {
+    if(err) {
+      return callback&&callback(err);
+    }
+    
+    if((quickReplyPayload in obj)) 
```

Comment:
Sir, use standard indentation. "if() {". Also, can you move this into a function, since we are repeating the same code?